presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/mysql/FunctionNamespaceDao.java

```diff
@@ -44,13 +44,15 @@
 public interface FunctionNamespaceDao
 {
     @SqlUpdate("CREATE TABLE IF NOT EXISTS <function_namespaces_table> (\n" +
+            "   id int(11) unsigned NOT NULL AUTO_INCREMENT,\n" +
```

Comment:
Why do we need this?

Reply:
Removed

Reply:
If you remove this, please also remove the primary key.